docs/source/tutorials/text.ipynb

```diff
@@ -24,6 +24,54 @@
     "- Train a more robust version of the same neural net via cleanlab's `CleanLearning` wrapper.\n"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-info\">\n",
+    "Quickstart\n",
+    "<br/>\n",
+    "    \n",
+    "Do you have an sklearn compatible `model`, text `data` and true `labels`? Run the code below to train your `model` and get label issues.\n",
+    "\n",
+    "\n",
+    "<div  class=markdown markdown=\"1\" style=\"background:white;margin:16px\">  \n",
+    "    \n",
+    "```python\n",
+    "\n",
+    "from cleanlab.classification import CleanLearning\n",
+    "\n",
+    "cl = CleanLearning(model)\n",
+    "_ = cl.fit(data, labels)\n",
+    "label_issues = cl.get_label_issues()\n",
+    "    \n",
+    "\n",
+    "```\n",
+    "    \n",
+    "</div>\n",
+    "    \n",
+    "Do you already have true `labels` and `pred_probs` from a trained model trained on text data? Run the code below to get ranked label issue indices.\n",
+    "\n",
+    "\n",
+    "<div  class=markdown markdown=\"1\" style=\"background:white;margin:16px\">  \n",
+    "    \n",
+    "```python\n",
+    "\n",
+    "from cleanlab.filter import find_label_issues\n",
+    "\n",
+    "ranked_label_issues = find_label_issues(\n",
+    "    labels,\n",
+    "    pred_probs,\n",
+    "    return_indices_ranked_by=\"self_confidence\",\n",
+    ")\n",
+    "    \n",
+    "\n",
+    "```\n",
+    "    \n",
+    "</div>\n",
```

Comment:
seems like a lot of newline chars, but I assume they are necessary for good formatting. Just noting in case they're not necessary

Reply:
Super necessary 😅